packages/cfd/src/Stores/Modules/CFD/cfd-store.js

```diff
@@ -296,10 +296,14 @@ export default class CFDStore extends BaseStore {
             if (!response.error) {
                 this.setError(false);
 
-                const trading_platform_accounts_list_response = await WS.tradingPlatformAccountsList(
-                    CFD_PLATFORMS.CTRADER
-                );
-                this.root_store.client.responseTradingPlatformAccountsList(trading_platform_accounts_list_response);
+                const account_list = {
+                    echo_req: response.echo_req,
```

Comment:
do we need to pass this?

Reply:
Yes exactly, echoing request should not be needed.

Reply:
We need this. `responseTradingPlatformAccountsList` is extracting platfrom from `echo_req`. 
<img width="797" alt="Screenshot 2024-06-28 at 1 11 45 PM" src="https://github.com/binary-com/deriv-app/assets/126637868/2542293d-1a14-4e12-9020-064ba84f330d">